hack/update-staging-godeps.sh

```diff
@@ -46,34 +46,35 @@ if [ "${SKIP_RESTORE}" != "true" ]; then
 	GOPATH="${godepRestoreDir}:${KUBE_ROOT}/staging:${ORIGINAL_GOPATH}"
 	# restore from kubernetes godeps to ensure we get the correct levels
 	# you get errors about the staging repos not using a known version control system
-	godep restore > ${godepRestoreDir}/godep-restore.log	2>&1 || true
+	godep restore > ${godepRestoreDir}/godep-restore.log 2>&1 || true
```

Comment:
I removed the `2>&1 || true` and it worked fine. In which case does one get errors?